rest_framework-stubs/viewsets.pyi

```diff
@@ -11,7 +11,7 @@ class ViewSetMixin(object):
     basename: str
     request: Any
     @classmethod
-    def as_view(cls, **initkwargs: Any) -> Callable: ...
+    def as_view(cls, actions: Optional[Dict[str, Any]] = None, **initkwargs: Any) -> Callable: ...
```

Comment:
Isn't it `Optional[Dict[str, str]]`?

Why `str, str`? 
Searching trough the docs gave me this impression: https://github.com/encode/django-rest-framework/blob/master/rest_framework/viewsets.py#L6

Can you please point to any other use-cases?

Reply:
Yes, it seems that it's a mapping between http method and method name on the class, it should be `Dict[str, str]` or maybe `Mapping[str, str]` to be a bit more general.

Reply:
@mkurnikov `Mapping` could be unnecessarily broad. IMO the type should be `Dict[str, str]` unless someone explicitly requests to change it.